ansible/roles/ocp4-workload-sso-cloud-native-apps/tasks/install-username-distribution.yaml

```diff
@@ -0,0 +1,76 @@
+---
+- name: search for username distribution tool
+  k8s_facts:
+    kind: DeploymentConfig
+    name: get-a-username
+    namespace: guides
+  register: r_gau_dc
+
+- name: deploy redis
```

Comment:
I'd prefer if the redis template was split up into individual YAML / YAML.j2 files and processed via `k8s` rather than `oc process`.

Reply:
this also needs a bunch of work also effects Quarkus, CCN, JDG workloads